prebid-mobile/prebid-mobile.md

```diff
@@ -11,31 +11,71 @@ nav_section: prebid-mobile
+{:toc }
 
-Benefits and features include:
+Prebid Mobile is an open-source library that provides an end-to-end header bidding solution for mobile app publishers. Use this library with your ad server's Mobile SDK to communicate with Prebid Server to request and receive bids over RTB. These bids can then compete directly with bids from your primary ad server.
 
-- Single integration point with Prebid Server with direct access to more mobile buyers
-- Server side configuration management for bidders and other settings, no need to update your app to make changes
-- Minimized latency due to background pre-caching of creatives
-- Increased yield due to increased competition between demand sources
-- A lightweight library that uses simple query strings to send ad requests and to pass results
+Prebid Mobile libraries are available for iOS and Android.
 
-### How it Works
+## Benefits and Features
 
-The Prebid Mobile SDK sends requests to Prebid Server to help app developers to access additional demand that is not available from their mobile ad network.
+Some of the benefits to using the Prebid Mobile header bidding solution include:
 
-This means you **must** have a Prebid Server account in order to use Prebid Mobile. To set up your Prebid Server account for Prebid Mobile, refer to [Get Started with Prebid Server]({{site.github.url}}/prebid-mobile/prebid-mobile-pbs.html)
+-   Provides a single integration point with Prebid Server, providing direct access to more mobile buyers.
+-   Allows for server-side configuration management; no need for developers to update the app to make configuration changes.
+-   Provides a transparent, open source header bidding solution.
+-   Flattens mediation layers, reducing ad ops burden for managing mediation partners.
+-   Reduces latency compared to mediation.
+-   Designed to integrate with Prebid Server, and can also integrate with any vendor hosting their own Prebid Server deployment (vendor must be registered in Prebid Mobile as a Prebid Server host).
 
- 1. As the Prebid Mobile module runs, it fetches bids from your demand partners integrated with Prebid Server.
- 2. Our Prebid Mobile module sends this bid information to your primary ad server SDK using key/value targeting.
- 3. The primary ad server SDK sends a request to the primary ad server including these custom parameters.  These parameters will match the key/value targeting previously configured on Prebid line items by your ad ops teams.
- 4. If the Prebid Mobile bid wins, the primary ad server returns our creative JS to your primary ad server SDK's ad view.
- 5. The creative JS in the ad view fetches the actual cached creative content from Prebid Server. When this happens, the impression is counted.
+## Requirements
+
+-   **Prebid Server Configuration**  
+    You *must* have a Prebid Server account in order to use Prebid Mobile. Prebid Server is a server-based host that communicates bid requests and responses between Prebid Mobile and demand partners.  
```

Comment:
This isn't providing information on Prebid Server Configurations, and seems like it might fit better in a "getting started" section.

Reply:
It's not supposed to explain how to configure it, it's just pointing out that in order to use prebid mobile they have to have a prebid server configuration, it won't work without prebid server.